Svg/Clipping and Masking/SvgClipPath.cs

```diff
@@ -54,7 +53,7 @@ public GraphicsPath GetClipRegionPath(SvgVisualElement owner)
                 this._pathDirty = false;
             }
 
-            var result = cachedClipPath;
+            var result = cachedClipPath.Clone();
```

Comment:
so one issue was that the same clippath-instance was transformed over and over again?

Reply:
yes thats why we got an arithmetic overflow.